crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -6,6 +6,13 @@ use bevy_core::{cast_slice, Pod};
 use copyless::VecHelper;
 use wgpu::BufferUsages;
 
+/// A `BufferVec` is a structure for storing data that has already been properly formatted
```

Comment:
```suggestion
/// A structure for storing raw bytes that has already been properly formatted
```
No need to repeat the struct name.

It's also helpful to call out the fact that this data is super raw.